greenpithumb/moisture_sensor.py

```diff
@@ -1,17 +1,44 @@
 class MoistureSensor(object):
     """Wrapper for DHT11 moisture sensor."""
 
-    def __init__(self, adc, channel):
+    def __init__(self, adc, pi_io, channel, gpio_pins, clock):
         """Creates a new MoistureSensor wrapper.
 
         Args:
             adc: ADC(analog to digital) interface to receive analog signals from
                 moisture sensor.
+            pi_io: Raspberry Pi I/O interface.
             channel: ADC channel to which the moisture sensor is connected.
+            gpio_pins: Tuple of length 2 of the Raspberry Pi GPIO pins that
```

Comment:
I feel like we might as well just make them two ints. Tuple would be more useful if it might eventually grow to > 2 pins, but this is a case where we know for sure it will always be 2. We save ourselves a variable here, but I think we add complexity because with a tuple, it can have a length != 2 and we crash, whereas the caller can't screw up two ints.